sdk/python/feast/infra/offline_stores/file.py

```diff
@@ -305,26 +404,29 @@ def evaluate_offline_job():
                 else [event_timestamp_column]
             )
 
-            source_df.sort_values(by=ts_columns, inplace=True)
+            # source_df = source_df.sort_values(by=ts_columns[0])
```

Comment:
what is happening here?

Reply:
I've commented this during profiling (I wanted to check how sorting affects the performance) and forgotten to uncomment.  Fixed - this sort is important because in the next step we ```drop_duplicates``` with ```keep="last"```. The dask limitation here is that we can sort only by one column in this case we have to sort by ```event_timestamp_column``` rather than ```[event_timestamp_column, created_timestamp_column]``` which is still correct.

Reply:
@qooba we currently use `created_timestamp_column` in order to deduplicate rows with duplicate event timestamps. We use the latest `created_timestamp_column` in that case. This is often useful in append only logs, since it allows a user to just reingest new rows with the same `event_timestamp`, but with fixed feature values. 

If we don't use created timestamp for sorting then we lose the guarantee of only returning the latest event_timestamp value by created_timestamp. Wouldn't it make more sense to first sort by `created_timestamp` and then `event_timestamp` in order to have determinism in the output and also consistency with our other offline store implementations?

Reply:
fixed

Reply:
Thanks, still some failures. Will wait for tests to pass then review :)

Reply:
Hmmm, those failures may be unrelated to you. We are looking into it.

Reply:
I fixed some related to my changes but still there is something with datetimes, anyway I will check :)